neutron-org/neutron-airdrop-transfer
# feat: airdrop transfer contract #ntrn-116

https://hadronlabs.atlassian.net/browse/NTRN-116

How to run integration tests:
1. in root: `make build`
2. cd integration-tests
3. run `yarn`
4. run `yarn test`
5. tests should pass!
